tensorbay/opendataset/RP2K/loader.py

```diff
@@ -57,7 +57,8 @@ def RP2K(path: str) -> Dataset:
                 continue
             image_paths = _glob(catagory_path, ("*.jpg", "*.png"))
             for image_path in image_paths:
-                data = Data(image_path)
+                remote_path = os.path.splitext(os.path.basename(image_path))[0].replace(" ", "_")
```

Comment:
commit message:
`fix(opendataset): correct the illegal remote paths for RP2K dataset`

Reply:
Why `splitext`?